README.md

```diff
@@ -1470,6 +1470,7 @@ API | Description | Auth | HTTPS | CORS |
 | [API-FOOTBALL](https://www.api-football.com/documentation-v3) | Get information about Football Leagues & Cups | `apiKey` | Yes | Yes |
 | [ApiMedic](https://apimedic.com/) | ApiMedic offers a medical symptom checker API primarily for patients | `apiKey` | Yes | Unknown |
 | [balldontlie](https://balldontlie.io) | Balldontlie provides access to stats data from the NBA | No | Yes | Yes |
+| [Bet365](https://rapidapi.com/b365api-b365api-default/api/betsapi2/) | Live results and fixtures on a variety of sports for betting | `apiKey` | Yes | Unknown |
```

Comment:
The API is mainly returning the responses related to events. 
So It's better to update the description according to it and please don't use the words like betting and fixture. Please polish the description a bit.

Thanks